### ISSUE TITLE
DB/Gossip: Eldara Dawnrunner

### ISSUE THREAD

**Issue by STARRHELD** [TrinityCore/TrinityCore#20640]
Description:
Eldara Dawnrunner is missing all of the creature_text

```
Do not fight the enemy unprepared, <name>. I can provide you with very powerful items... provided you have proven yourself to us.

Eldara Dawnrunner says: <name>, it's very good to see you again!
Eldara Dawnrunner says: <name>, one of these days I want to buy you a drink.
Eldara Dawnrunner says: <name>, thank you again for helping us. We couldn't have achieved any of this without you.
Eldara Dawnrunner says: <name>, you have no idea what a relief it is to see you!
Eldara Dawnrunner says: Dawnblade, demons and naga beware, <name> is back.
Eldara Dawnrunner says: Everyone look, it's <name>!
Eldara Dawnrunner says: Hey, <name>, try to leave some of the glory for us, would ya?
Eldara Dawnrunner says: How many wretched do you think <name> has put out of their misery?
Eldara Dawnrunner says: I don't even want to think about what'll happen here if <name> ever decides not to come back.
Eldara Dawnrunner says: I guarantee that without <name>, this whole offensive would have been a failure.
Eldara Dawnrunner says: If I were one of those Dawnblade blood elves, I'd be swimming away from the island now that <name> is here.
Eldara Dawnrunner says: If it weren't for <name>, we'd all be long dead.
Eldara Dawnrunner says: It's an honor to serve with you, <name>.
Eldara Dawnrunner says: Lay waste to the demons on the Dead Scar lately, <name>?
Eldara Dawnrunner says: Milady, <name>, we are fortunate to have you here with us.
Eldara Dawnrunner says: Milord, <name>, we are fortunate to have you here with us.
Eldara Dawnrunner says: No matter what, your efforts will always be exalted here, <name>!
Eldara Dawnrunner says: Snap to attention, <name>, one of our finest, is here!
Eldara Dawnrunner says: The Shattered Sun Offensive is lucky to have <name> on our side.
Eldara Dawnrunner says: There's our hero right there! How are you doing, <name>?
Eldara Dawnrunner says: There's our heroine right there! How are you doing, <name>?
Eldara Dawnrunner says: Three cheers for <name>!
Eldara Dawnrunner says: Welcome back, <name>.
Eldara Dawnrunner says: What a coincidence... we were just talking about you, <name>!
Eldara Dawnrunner says: Who needs sentries? <name> is here!
```

https://wow.gamepedia.com/Eldara_Dawnrunner

Branch(es):

3.3.5

TC rev. hash/commit:

TrinityCore rev. 9fa6885 2017-08-08 11:36:46 -0300 (3.3.5 branch) (Win64, Release, Static) (worldserver-daemon)

TDB version: 335.63

Operating system: Windows 7

**Comment by STARRHELD** [comment]
To save someone the headache:

```SQL
SELECT * FROM broadcast_text WHERE ID IN (24086, 24087, 24088, 24089, 24098, 24099, 24100, 24102, 24103, 24106, 24107, 24108, 24110, 24111, 24112, 24113, 24115, 24116, 24119, 24120, 24121, 24122, 24302);
```
```
| ID    | Language | MaleText                                                                                                | FemaleText                                                                                                                     | EmoteID0 | EmoteID1 | EmoteID2 | EmoteDelay0 | EmoteDelay1 | EmoteDelay2 | SoundId | Unk1 | Unk2 | VerifiedBuild |
|-------|----------|---------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------|----------|----------|----------|-------------|-------------|-------------|---------|------|------|---------------|
| 24086 | 0        | $n, it's very good to see you again!                                                                    | $n, it's very good to see you again!                                                                                           | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24087 | 0        | Everyone look, it's $n!                                                                                 | Everyone look, it's $n!                                                                                                        | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24088 | 0        | Dawnblade, demons and naga beware, $n is back.                                                          | Dawnblade, demons and naga beware, $n is back.                                                                                 | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24089 | 0        | $G Milord : Milady;, $n, we are fortunate to have you here with us.                                     | $G Milord : Milady;, $n, we are fortunate to have you here with us.                                                            | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24098 | 0        | The Shattered Sun Offensive is lucky to have $n on our side.                                            | The Shattered Sun Offensive is lucky to have $n on our side.                                                                   | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24099 | 0        | If it weren't for $n, we'd all be long dead.                                                            | If it weren't for $n, we'd all be long dead.                                                                                   | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24100 | 0        | $n, thank you again for helping us. We couldn't have achieved any of this without you.                  | $n, thank you again for helping us. We couldn't have achieved any of this without you.                                         | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24102 | 0        | Three cheers for $n!                                                                                    | Three cheers for $n!                                                                                                           | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24103 | 0        | It's an honor to serve with you, $n.                                                                    | It's an honor to serve with you, $n.                                                                                           | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24106 | 0        | If I were one of those Dawnblade blood elves, I'd be swimming away from the island now that $n is here. | If I were one of those Dawnblade blood elves, I'd be swimming away from the island now that $n is here.                        | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24107 | 0        | How many wretched do you think $n has put out of their misery?                                          | How many wretched do you think $n has put out of their misery?                                                                 | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24108 | 0        | Snap to attention, $n, one of our finest, is here!                                                      | Snap to attention, $n, one of our finest, is here!                                                                             | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24110 | 0        | Welcome back, $n.                                                                                       | Welcome back, $n.                                                                                                              | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24111 | 0        | There's our $g hero : heroine; right there! How are you doing, $n?                                      | There's our $g hero : heroine; right there! How are you doing, $n?                                                             | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24112 | 0        | I guarantee that without $n, this whole offensive would have been a failure.                            | I guarantee that without $n, this whole offensive would have been a failure.                                                   | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24113 | 0        | No matter what, your efforts will always be exalted here, $n!                                           | No matter what, your efforts will always be exalted here, $n!                                                                  | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24115 | 0        | Lay waste to the demons on the Dead Scar lately, $n?                                                    | Lay waste to the demons on the Dead Scar lately, $n?                                                                           | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24116 | 0        | $n, one of these days I want to buy you a drink.                                                        | $n, one of these days I want to buy you a drink.                                                                               | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24119 | 0        | $n, you have no idea what a relief it is to see you!                                                    | $n, you have no idea what a relief it is to see you!                                                                           | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24120 | 0        | What a coincidence... we were just talking about you, $n!                                               | What a coincidence... we were just talking about you, $n!                                                                      | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24121 | 0        | Who needs sentries? $n is here!                                                                         | Who needs sentries? $n is here!                                                                                                | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24122 | 0        | Hey, $n, try to leave some of the glory for us, would ya?                                               | Hey, $n, try to leave some of the glory for us, would ya?                                                                      | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
| 24302 | 0        |                                                                                                         | Do not fight the enemy unprepared, $n.  I can provide you with very powerful items... provided you have proven yourself to us. | 0        | 0        | 0        | 0           | 0           | 0           | 0       | 0    | 1    | 18019         |
```

**Comment by tkrokli** [comment]
The interesting part will be to find out how those lines are triggered, how many (one or more than one) line(s) are said each time and how often.

**Comment by STARRHELD** [comment]
I agree, going to be a good one.

**Comment by STARRHELD** [comment]
> Something to start with.
```SQL
-- Eldara Dawnrunner
DELETE FROM `creature_text` WHERE `CreatureID`=25032;
INSERT INTO `creature_text` (`CreatureID`,`GroupID`,`ID`,`Text`,`Type`,`Language`,`Probability`,`Emote`,`Duration`,`Sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
(25032,0,0,"$n, it's very good to see you again!",         								  																				 12,0,100,0,0,0,24086,0,'Eldara Dawnrunner'),
(25032,0,0,"Everyone look, it's $n!",           											  							                  							 12,0,100,0,0,0,24087,0,'Eldara Dawnrunner'),
(25032,0,0,"Dawnblade, demons and naga beware, $n is back.",                       									            							 12,0,100,0,0,0,24088,0,'Eldara Dawnrunner'),
(25032,0,0,"$G Milord : Milady;, $n, we are fortunate to have you here with us.", 						  														 12,0,100,0,0,0,24089,0,'Eldara Dawnrunner'),
(25032,0,0,"The Shattered Sun Offensive is lucky to have $n on our side.",        				 		  														 12,0,100,0,0,0,24098,0,'Eldara Dawnrunner'),
(25032,0,0,"If it weren't for $n, we'd all be long dead.",                                           														 12,0,100,0,0,0,24099,0,'Eldara Dawnrunner'),
(25032,0,0,"$n, thank you again for helping us. We couldn't have achieved any of this without you.", 														 12,0,100,0,0,0,24100,0,'Eldara Dawnrunner'),
(25032,0,0,"$Three cheers for $n!",                                                                  						 								 12,0,100,0,0,0,24102,0,'Eldara Dawnrunner'),
(25032,0,0,"It's an honor to serve with you, $n.",                                                   						   							 12,0,100,0,0,0,24103,0,'Eldara Dawnrunner'),
(25032,0,0,"If I were one of those Dawnblade blood elves, I'd be swimming away from the island now that $n is here.",   							 12,0,100,0,0,0,24106,0,'Eldara Dawnrunner'),
(25032,0,0,"How many wretched do you think $n has put out of their misery?",    																					 12,0,100,0,0,0,24107,0,'Eldara Dawnrunner'),
(25032,0,0,"Snap to attention, $n, one of our finest, is here!",    																									 12,0,100,0,0,0,24108,0,'Eldara Dawnrunner'),
(25032,0,0,"Welcome back, $n.",    																													 							 12,0,100,0,0,0,24110,0,'Eldara Dawnrunner'),
(25032,0,0,"There's our $g hero : heroine; right there! How are you doing, $n?",   											       							 12,0,100,0,0,0,24111,0,'Eldara Dawnrunner'),
(25032,0,0,"I guarantee that without $n, this whole offensive would have been a failure.",   									                      12,0,100,0,0,0,24112,0,'Eldara Dawnrunner'),
(25032,0,0,"No matter what, your efforts will always be exalted here, $n!",   											          							 12,0,100,0,0,0,24113,0,'Eldara Dawnrunner'),
(25032,0,0,"Lay waste to the demons on the Dead Scar lately, $n?",   											                   							 12,0,100,0,0,0,24115,0,'Eldara Dawnrunner'),
(25032,0,0,"$n, one of these days I want to buy you a drink.",   											                         							 12,0,100,0,0,0,24116,0,'Eldara Dawnrunner'),
(25032,0,0,"$n, you have no idea what a relief it is to see you!",   											                         					 12,0,100,0,0,0,24119,0,'Eldara Dawnrunner'),
(25032,0,0,"What a coincidence... we were just talking about you, $n!",   											                         				 12,0,100,0,0,0,24120,0,'Eldara Dawnrunner'),
(25032,0,0,"Who needs sentries? $n is here!",   											                         												 12,0,100,0,0,0,24121,0,'Eldara Dawnrunner'),
(25032,0,0,"Hey, $n, try to leave some of the glory for us, would ya?",   											                                     12,0,100,0,0,0,24122,0,'Eldara Dawnrunner'),
(25032,0,0,"Do not fight the enemy unprepared, $n.  I can provide you with very powerful items... provided you have proven yourself to us.",  12,0,100,0,0,0,24302,0,'Eldara Dawnrunner');
```

**Comment by offl** [comment]
Maybe it was removed after 2.4.3. She says nothing [here](https://www.youtube.com/watch?v=ry9gCTpuBw4), [here](https://www.youtube.com/watch?v=I38M0UgPecU) and [here](https://www.youtube.com/watch?v=mG9x3SkzIJg).
You can see text [on this screenshot](http://kaylarin.com/islefun/WoWScrnShot_051808_193323.jpg), but [as author said](http://kaylarin.com/pre302screenshots.html) it was done before WotLK.

> The interesting part will be to find out how those lines are triggered, how many (one or more than one) line(s) are said each time and how often.

LOS OOC maybe.

**Comment by STARRHELD** [comment]
@offl Isn't possible to check Live realms if it's still in there? I mean this issue report is very very low priority.

**Comment by tkrokli** [comment]
> I'm not sure, but maybe it was removed after 2.4.3. She says nothing here, here and here.

Those 3 videos are either level 100 (GitGudGuides) or uploaded in 2017 and very short sequences as well. I don't think they are enough to tell whether or not those texts were removed in Mists of Pandaria or Warlords of Draenor. Maybe I have overlooked something, but is it clear in those videos that the player has got enough reputation with the Sha'tar to trigger the texts?
